src/main/groovy/si/dlabs/gradle/extensions/FindbugsExtension.groovy

```diff
@@ -7,8 +7,8 @@ import org.gradle.api.tasks.Input
  */
 class FindbugsExtension extends PluginBaseExtension {
 
-    @Input String reportFile = "reports/checkstyle/checkstyle.xml"
     @Input String effort = "max"
     @Input String reportLevel = "low"
+    @Input String reportType = "html"
```

Comment:
FindBugs itself only supports *either* html *or* xml reports as I've mentioned [here](https://github.com/sschuberth/soter/commit/2b2acff6215a0d1e6375d267835adb176132638d#diff-3036f7cd3d4a7d95903bcf5d03209ba5R204), so there's no need for an array here.